### PR TITLE
Enable Reader online tests

### PR DESCRIPTION
**Pull request by Maks3w**

### CODE REVIEW

**Comment by Maks3w** [comment]
Most of this tests could be refactored as unit tests.

File: test/Reader/ReaderTest.php [hunk 1]
```diff
@@ -158,7 +158,7 @@ public function testImportsUriAndThrowsExceptionIfNotAFeed()
             $this->markTestSkipped('testImportsUri() requires a network connection');
         }
 
-        Reader\Reader::import('http://twitter.com/alganet');
+        Reader\Reader::import('http://example.com');
```
Comment (weierophinney) [inline test/Reader/ReaderTest.php:161]: Why the change in URL? When I ran these tests last week, the original URLs worked correctly for the test expectations...

Comment (Maks3w) [inline test/Reader/ReaderTest.php:161]: Troubles with redirect to HTTPS protocol. I didn't want to deal with that https://travis-ci.org/zendframework/zend-feed/jobs/92411760#L317

**Comment by Maks3w** [comment]
Rest of changes was done because the source of the reference webpages has changed.

In the future this tests should be refactored with a mock of the http client.

**Comment by weierophinney** [comment]
Merged to develop for release with 2.6.0.
